scene/gui/progress_bar.h

```diff
@@ -43,11 +43,26 @@ class ProgressBar : public Range {
 	static void _bind_methods();
 
 public:
+	enum FillMode {
+		FILL_LEFT_TO_RIGHT = 0,
+		FILL_RIGHT_TO_LEFT,
+		FILL_TOP_TO_BOTTOM,
+		FILL_BOTTOM_TO_TOP
```

Comment:
```suggestion
		FILL_BOTTOM_TO_TOP
		FILL_MODE_MAX
```

Reply:
Wouldn't it make `FILL_MODE_MAX` be available to choose by the user in the editor's dropdown though?
And you've missed a comma.

Reply:
It wouldn't actually. As far as I know, the drop down options are added by using ADD_PROPERTY in the code. They're not generated automatically from enums. BTW I'm pretty new to contributing to Godot. What do I do about these requested changes? Do I need to commit them (as a big commit button appears below each change when I use the GitHub app)? Or I just amend my own commit according to these requests?

Reply:
> Do I need to commit them (as a big commit button appears below each change when I use the GitHub app)? Or I just amend my own commit according to these requests?

Either of those options is fine, but if you do the former, you will have to squash the commits into a single commit: https://docs.godotengine.org/en/stable/community/contributing/pr_workflow.html#the-interactive-rebase

Reply:
> It wouldn't actually. As far as I know, the drop down options are added by using ADD_PROPERTY in the code. They're not generated automatically from enums.

You're right, I've somehow missed that. So what @EricEzaM suggested is the way to go.